src/dash/models/DashManifestModel.js

```diff
@@ -292,6 +292,24 @@ function DashManifestModel(config) {
         return adaptation[DashConstants.CENC_DEFAULT_KID];
     }
 
+    function getLabel(adaptation) {
```

Comment:
To keep consistency with rest of similar methods I think this should be named getLabelForAdaptation

Reply:
@epiclabsDASH Updated!